src/jdk.jartool/share/man/jar.md

```diff
@@ -213,6 +217,26 @@ operation modes:
 `--version`
 :   Prints the program version.
 
+## Integrity of a jar Archive
+As a jar archive is based on ZIP format, it is possible to create a jar archive using tools
+other than the `jar` command. The --validate option may be used to perform the following
+integrity checks against a jar archive:
+
+- That there are no duplicate Zip Entry file names
+- Verify that the Zip Entry file name:
+    - is not an absolute path
+    - the file name is not '.' or '..'
+    - does not contain a backslash, '\\'
+    - does not contain a drive letter
+    - path element does not include '.' or '..
+- The API exported by a multi-release jar archive is consistent across all different release
+  versions.
+
+The jar tool returns a status code of 0 if there were no integrity issues encountered, otherwise
+a non-zero status code would be returned.
+When an integrity issue is reported, it will often require that the jar file is re-created by the
```

Comment:
Nit - "... require that the JAR file is re-created by the original source of the JAR file."